tests/integration/query/simple/with_filter/with_not_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
todo: Whilst they are a pain to write, I do think it is worth adding at least one schema test.  Particularly given the order bug Keenan found today.

Reply:
What do you mean by a schema test?

Reply:
If so I don't think this PR has any impact on those.

Reply:
> If so I don't think this PR has any impact on those.

It does, as the feature will actually work now, so we should ensure that it stays working.  I had a look though, and `_not` is already covered by them.